docs/pages/reference/authentication.mdx

```diff
@@ -121,6 +121,50 @@ $ tctl create -f cap.yaml
 </TabItem>
 </Tabs>
 
+### Local user locks
+
+A lock is placed on a local user if, within a 30 minute window, a local user has:
```

Comment:
Should we call this something else? It sounds a lot like we're referring to Teleport's lock resource, but this is something else.

Reply:
Please see update